app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<section>
+    <%= form_with model: [@post, @post.comments.build], method: :post, local: true do |form| %>
```

Comment:
Please add this form to `views/comments/new.html.erb` so it is shown after getting the new request from the comments controller.